data/mods/Magiclysm/mutations/mutations.json

```diff
@@ -8,5 +8,596 @@
+      "strength_damage": { "damage_type": "stab", "amount": 0.25 }
+    },
+    "spells_learned": [ [ "acid_bite", 1 ] ],
+    "prereqs2": [ "DRAGON_MAGIC" ]
+  },
+  {
+    "type": "mutation",
+    "id": "VOMITOUS",
+    "copy-from": "VOMITOUS",
+    "extend": { "category": [ "DRAGON_BLACK" ] },
+    "leads_to": [ "DRAGON_BREATH_BLACK", "OVERLY_ACID_GUT" ]
+  },
+  {
+    "type": "mutation",
+    "id": "OVERLY_ACID_GUT",
+    "name": { "str": "Overly Acidic Digestion" },
+    "description": "Your digestive issues have developed to become overly acidic which greatly impacts your ability to normally digest food, resulting in getting far less nutrition from the food you eat and making your need to eat more often.",
+    "points": -3,
+    "prereqs": [ "VOMITOUS" ],
+    "category": [ "DRAGON_BLACK" ],
+    "metabolism_modifier": 0.5,
+    "vitamins_absorb_multi": [ [ "all", [ [ "vitA", 0.5 ], [ "vitB", 0.5 ], [ "vitC", 0.5 ], [ "calcium", 0.5 ], [ "iron", 0.5 ] ] ] ]
+  },
+  {
+    "type": "mutation",
+    "id": "DRAGON_BREATH_BLACK",
+    "name": { "str": "Black Dragon Breath" },
+    "description": "Your digestive issues may not have settled entirely, but you can somewhat control them and now you have the ability to spit acid!",
+    "points": 2,
+    "prereqs": [ "VOMITOUS" ],
+    "prereqs2": [ "DRAGON_MAW_BLACK" ],
+    "category": [ "DRAGON_BLACK" ],
+    "threshreq": [ "THRESH_DRAGON_BLACK" ],
+    "spells_learned": [ [ "dragon_spit_acid", 1 ], [ "dragon_breath_acid", 1 ] ]
+  },
+  {
+    "type": "mutation",
+    "id": "THIRST",
+    "copy-from": "THIRST",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "THIRST2",
+    "copy-from": "THIRST2",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "CARNIVORE",
+    "copy-from": "CARNIVORE",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "DRAGONBLOOD",
+    "name": { "str": "Draconic Heat Dependence" },
+    "points": -1,
+    "description": "Your muscle response is dependent on ambient temperatures.  You lose 1% of your speed for every 5 (2.8) degrees below 65 F (18.3 C).  Unfortunately, this doesn't seem to help you conserve any energy.",
+    "changes_to": [ "DRAGONBLOOD2" ],
+    "types": [ "METABOLISM" ],
+    "category": [ "DRAGON_BLACK" ],
+    "temperature_speed_modifier": 0.2
+  },
+  {
+    "type": "mutation",
+    "id": "DRAGONBLOOD2",
+    "name": { "str": "High Draconic Heat Dependence" },
+    "points": -2,
+    "description": "Your muscle response is highly dependent on ambient temperatures.  You lose 1% of your speed for every 3 (1.7) degrees below 65 F (18.3 C).  Unfortunately, it seems you have to exert more energy to keep moving.",
+    "changes_to": [ "DRAGONBLOOD3" ],
+    "types": [ "METABOLISM" ],
+    "category": [ "DRAGON_BLACK" ],
+    "temperature_speed_modifier": 0.2,
+    "metabolism_modifier": 0.25
```

Comment:
"exert more energy to keep moving" i think this is backwards.

Reply:
My intent is to have the dragon type of cold blooded traits be wholly bad, make you move slower while cold AND require you to eat more, indicating that you need to use more energy to keep moving. I mirrored the metabolism modifier from the fast metabolism traits as well as the movement speed modifiers from the cold blooded traits. Is this incorrectly implemented?

Reply:
i have not been able to confirm, but what i was saying is that i thought the value 0.25 was x0.25 so 1/4 of the metabolic speed, rather than speeding it up. if you were able to confirm that it would be great. i'll pull down this pr one day this weekend for a full checkup on the mutations in game.

Reply:
Positive values of "metabolism_modifier" increase metabolism, and negative values decrease it. From JSON_INFO:
```
"metabolism_modifier": 0.333, // Extra metabolism rate multiplier. 1.0 doubles usage, -0.5 halves.
```